src/core/triggers.systemd.in

```diff
@@ -69,3 +70,90 @@ if posix.access("%{_localstatedir}/lib/rpm-state/systemd/needs-reload") then
         posix.wait(pid)
     end
 end
+
+%transfiletriggerin -P 100700 -p <lua> -- @sysusersdir@
+-- This script will process files installed in sysusers.d to
+-- create specified users automatically. The priority is set
+-- such that it will run before the tmpfiles file trigger.
+if posix.access("/run/systemd/system") then
+    pid = posix.fork()
+    if pid == 0 then
+        assert(posix.exec("%{_bindir}/systemd-sysusers"))
```

Comment:
any reason to not use `os.execute()`?

Reply:
I was following what the other triggers in this file do.

Reply:
There was some good reason for that, I don't remember though.